svgnative/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ endif()
 set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 project(SVGNativeViewer)
 enable_language(CXX)
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
This should only be done when Skia is enabled. Also, I wonder if we didn't make the change earlier? I thought that we did.

Reply:
Yes, I also just checked. You guys did make some of the changes here. My branch was a few commits behind main. :laughing:  Fixing it.

Reply:
Do you think this can be done in the `setting for Skia` block? Or should we NOT set the CXX standard if `CMAKE_TOOLCHAIN_FILE` exists?

Reply:
In the Skia settings should be fine. In all other cases we should build with 11.